test/JDBC/upgrade/latest/schedule

```diff
@@ -511,3 +510,5 @@ BABEL-730
 babel-4475
 babel-3254
 babel-4517
+babelfish_inconsistent_metadata
+1_grant_schema
```

Comment:
1. Why do we need separate test file? Can't we add new test cases into the existing GRANT_SCHEMA-vu* file? We should try to minimize the number of test files unless absolutely needed.
2. What's the use of GRANT_SCHEMA.sql test? Or what's the difference b/w GRANT_SCHEMA-vu* and GRANT_SCHEMA.sql?
3. Here we just have changed the latter casing for the new test file which will make it quite difficult for future developers to understand the difference b/w GRANT_SCHEMA-vu* and 1_grant_schema-vu* test files. On a high level, it looks like what we want is to have GRANT_SCHEMA-before-15_7-16_3-vu* and GRANT_SCHEMA-vu* instead.
4. Why have we missed 15_6 and 15_5 schedule files?
5. Let's also add tests for database-level dump restore for this new catalog table as we discussed offline.

Reply:
1, 2 and 3: GRANT_SCHEMA.mix has all the various scenarios mentioned in the test plan. All of it can't be written in vu-* format. We have GRANT_SCHEMA-before-15_7-16_3-vu*  and 1_GRANT_SCHEMA-vu* for upgrade testing of permissions and catalog change. We have added [1_] because we wanted this test to run in the beginning since some permissions in other test files were interfering with these tests.
4. Added.
5. Added.